README.md

```diff
@@ -16,9 +16,40 @@ In summary, our software suite addresses the multifaceted challenges associated
 
 ## Results
 ### NWBInspector
-It can be time-consuming and error-prone to manually inspect all of the NWB files in a dataset, especially as datasets grow very large. DANDI datasets frequently occupy hundreds of GBs to TBs of disk space, spread across thousands of individual NWB files. This challenge is compounded for new users who may lack familiarity with NWB best practices, further hampering the effectiveness of manual inspection.
+It can be time-consuming and error-prone to manually inspect all of the NWB files in a dataset, especially as datasets grow very large. DANDI datasets frequently occupy hundreds of GBs to TBs of disk space, spread across thousands of individual NWB files [cite DANDI]. This challenge is compounded for new users who may lack familiarity with NWB best practices, further hampering the effectiveness of manual inspection. To address this problem, we have developed _NWB Inspector_, a Python-based package that assesses the quality of NWB files based on their compliance with established best practices.
 
-...
+After installation, _NWBInspector_ can be used very easily from the command line. Simply run
+```bash
+    nwbinspector path/to/my/data.nwb
+```
+to inspect a single NWB file or
+```bash
+    nwbinspector path/to/my/data/dir/
+```
+to inspect all the NWB files in a given directory.
 
-TBD
+This command will produce a report of all the issues flagged by _NWB Inspector_ directly in the terminal, ordered by importance as shown below.
+[Example output]
 
+Unlike the _PyNWB_ validator, which checks for strict schema compliance, _NWBInspector_ only serves as an automatic data review aid. This means that, in an effort to catch as many potential mistakes as possible, _NWBInspector_ may raise warnings for intentional deviations from standard best practices. Conversely, if a user makes a mistake that does not violate any established best practices and it passes the _PyNWB_ validator, _NWBInspector_ might not catch that mistake. This structure leads to a trade-off between comprehensive inspection, which catches as many potential mistakes as possible, and an unobtrusive inspection process that only highlights truly problematic choices rather than intentional but unorthodox decisions.
+
+Our solution to this trade-off involves splitting the results into three tiers of warnings/errors. The first tier is for best practice violations, which are clear violations of known best practices, such as negative spike times. The second tier is for critical issues, which are potentially critical mistakes, such as zero-rate timestamps. The third tier is for best practice suggestions, which are recommendations for improving the NWB file, such as adding an informative description. This approach allows our inspection process to be as comprehensive as possible while minimizing disruptions from ignoring specific best practice suggestions.
```

Comment:
Critical issues come before violations

Critical issues are essentially invalidations that are too complicated to define in the schema language or even in PyNWB logic. They can also be heuristical checks of invalid fields that can't be identified deterministically.